lib/app/views/state_status.erb

```diff
@@ -7,17 +7,16 @@
       </div>
     </div>
 
-    <h4>Conformance Statement Resource Support</h4>
 
-    <% if instance.supported_resources.count %>
-      <ul>
+    <% if instance.supported_resources.count > 0%>
+    <h4>Conformance Statement Resource Support</h4>
+    <ul>
       <% instance.supported_resources.each do |resource| %>
         <% if resource.supported %>
           <li><%= resource.resource_type %> [
               <% if resource.read_supported %> read, <% end %>
               <% if resource.vread_supported %> vread, <% end %>
-              <% if resource.history_supported %> history, <% end %>
-              <% if resource.scope_supported %> authorized <% end %>
+              <% if resource.history_supported %> history<% end %><% if resource.scope_supported %>, authorized <% end %>
```

Comment:
Hmm, this will result in weird behavior if vread and scope are supported but not history.  I think we should add a method to the resource model that is something like 'supported_interactions', which returns an array and we can just `join(',')` out here.

Reply:
I like that idea, hopefully you like the implementation